test/services/fetchers/UniswapV3MultiFetcher.test.ts

```diff
@@ -0,0 +1,98 @@
+import chai from 'chai';
+
+import UniswapV3MultiFetcher, {
+  UniswapV3MultiFetcherParams,
+} from '../../../src/services/fetchers/UniswapV3MultiFetcher.js';
+import Settings from '../../../src/types/Settings.js';
+import {getTestContainer} from '../../helpers/getTestContainer.js';
+import {BigNumber} from 'ethers';
+import {ContractHelperRepository} from '../../../src/repositories/ContractHelperRepository.js';
+import sinon from 'sinon';
+import {ContractHelper} from '../../../src/services/fetcherHelper/contracts/ContractHelper.js';
+import {DexRepository} from '../../../src/repositories/DexRepository.js';
+import {ChainsIds} from '../../../src/types/ChainsIds.js';
+import {DexProtocolName} from '../../../src/types/DexProtocolName.js';
+
+const {expect} = chai;
```

Comment:
```suggestion
const {expect} = chai;
import sinon from 'sinon';
import {BigNumber} from 'ethers';

import UniswapV3MultiFetcher, {
  UniswapV3MultiFetcherParams,
} from '../../../src/services/fetchers/UniswapV3MultiFetcher.js';
import Settings from '../../../src/types/Settings.js';
import {getTestContainer} from '../../helpers/getTestContainer.js';
import {ContractHelperRepository} from '../../../src/repositories/ContractHelperRepository.js';
import {ContractHelper} from '../../../src/services/fetcherHelper/contracts/ContractHelper.js';
import {DexRepository} from '../../../src/repositories/DexRepository.js';
import {ChainsIds} from '../../../src/types/ChainsIds.js';
import {DexProtocolName} from '../../../src/types/DexProtocolName.js';
```

I removed `chain` but I am not sure.

Reply:
It throws an error: ReferenceError: chai is not defined